frontend/src/metabase/public/components/widgets/AdvancedEmbedPane.jsx

```diff
@@ -74,7 +74,11 @@ const AdvancedEmbedPane = ({
         onChange={() => onChangePane(pane === "preview" ? "code" : "preview")}
       />
       {pane === "preview" ? (
-        <PreviewPane className="flex-full" previewUrl={iframeUrl} />
+        <PreviewPane
+          className="flex-full"
+          previewUrl={iframeUrl}
+          isTransparent={displayOptions.theme === "transparent"}
```

Comment:
pass a prop to determine if we want to set background to checkerboard pattern. I originally always set the background to checkerboard pattern but it looks weird when changing themes. e.g. When I change from `light` to `dark` I would briefly see the checkerboard pattern before the whole iframe is loaded.